Services/Export/classes/class.ilExportGUI.php

```diff
@@ -271,7 +271,7 @@ public function createExportFile(): void
                 } elseif ($this->getParentGUI() instanceof ilContainerGUI) {
                     $this->showItemSelection();
                     return;
-                } elseif ($format == "xml") {        // standard procedure
+                } elseif ($format == "xml" || $format === 'xmlres') {        // standard procedure
```

Comment:
This introduces a dependency in the `Export` component to a concrete identifier of an export provider (here: the `Test`).

Reply:
Yes, also not happy with this

Reply:
Please add the coding authorities for the `Export` component to decide upon the required changes as well.

Reply:
Cannot edit assignees myself. Insufficient rights
That would be @smeyer-ilias right?